modules/deployment-plug-in-cluster.adoc

```diff
@@ -10,14 +10,14 @@ After pushing an image with your changes to a registry, you can deploy the plug-
 
 .Procedure
 
-. To deploy your plug-in to a cluster, instantiate the link:https://github.com/spadgett/console-plugin-template/blob/main/template.yaml[template] by running the following command:
+. To deploy your plug-in to a cluster, instantiate your plug-in by running the following command:
 +
 [source,terminal]
 ----
 $ oc process -f template.yaml \
   -p PLUGIN_NAME=my-plugin \ <1>
   -p NAMESPACE=my-plugin-namespace \ <2>
-  -p IMAGE=quay.io/my-repository/my-plugin:latest \ <3>
+  -p IMAGE=quay.io/my-repository/my-plugin:release-4.10 \ <3>
```

Comment:
@spadgett Hey Sam! I feel like this may not be correct.  Is adding `release-4.10` correct here? 

I also realized we still have your template linked, so I removed it to match what 4.11 docs states (before the Helm updates).

Reply:
I would leave it `latest`.

This can be any tag you choose when you build your image. `latest` is typically used for the latest version or if you don't specify a tag, so it's a good example.